js/lightbeam.js

```diff
@@ -236,7 +236,7 @@ const lightbeam = {
       this.websites[data.hostname] = data;
       this.updateVars(data.firstParty);
     }
-    if (!data.firstParty) {
+    if (data.firstPartyHostnames) {
```

Comment:
Same goes for here. We just want to know what the new links are, if any and the `firstParty` boolean doesn't reflect that.